core/store/src/trie/mem/updating.rs

```diff
@@ -1668,9 +1745,9 @@ mod tests {
 
         for (key, value) in changes {
             if let Some(value) = value {
-                update.insert_memtrie_only(&key, FlatStateValue::on_disk(&value));
+                update.insert_memtrie_only(&key, FlatStateValue::on_disk(&value)).unwrap();
             } else {
-                update.delete(&key);
+                update.delete(&key).unwrap();
```

Comment:
I'm not sure this path is covered in tests, can you confirm?

Reply:
This is not great, but result of `pub fn delete(&mut self, key: &[u8]) -> Result<(), StorageError> {` for memtrie is always Ok.

I did it to make interface the same, as TrieStorage is expected to return an Error. We can consider improving the interface later.